qemu/tests/cfg/balloon_service.cfg

```diff
@@ -22,8 +22,8 @@
         run_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -r"
     repeat_times = 5
     base_path = "/machine/peripheral/"
-    set_balloon_property = "guest-stats-polling-interval"
-    get_balloon_property = "guest-stats"
+    polling_property = "guest-stats-polling-interval"
+    stats_property = "guest-stats"
```

Comment:
@vivianQizhu  These two variants are just two different properties, they both can be set and get.  I think it is clearer to change the name. Thanks.

Reply:
These two command, one is used to get/set the "interval", one is used to get "stats".
How about `polling_interval_property` and `polling_stats_property` ?

Reply:
@yanan-fu You are right about the usage of them. But I don't see the point in changing them. And I would like to keep the variant name short if possible.  Thanks!

Reply:
@huangyum That's both okay for me, but what do you guys think of "polling_interval" and "polling_stats", because "property" actually seems a little superfluous...

Reply:
@vivianQizhu Actually "polling_interval" has been used to present the value of the interval. These two are just the property names, they don't matter much.  I would like to keep them if nobody are strongly against them or have better idea. Thanks!